GoogleForCreators/web-stories-wp
# Settings: Improve Site Kit detection logic

## Summary

The google analytics input is only disabled when the site analytics module is installed and active. Otherwise, the input is enabled.

## Relevant Technical Choices

- the input will only be disabled when the site kit plugin is installed and activated.

## To-do

none

## User-facing changes

- the input is disabled when the site kit plugin is installed and activated. It used to be disabled if the site kit plugin was just installed.

## Testing Instructions

- start out with `Site Kit` plugin not installed
- go to stories dashboard settings. Should see text prompting you to install the plugin.
- add `Site Kit` plugin from google to wordpress
- go to stories dashboard settings. Should see text prompting you to activate the plugin
- You can't activate the plugin in a staging environment, but if you did you'd see the 3rd screenshot below:

|||
|--|--|
|No Site Kit|![image](https://user-images.githubusercontent.com/22185279/104629431-bcd23100-5656-11eb-9297-cfbc7b6fdd0c.png)|
|Site Kit Installed|![image](https://user-images.githubusercontent.com/22185279/104629349-9e6c3580-5656-11eb-9470-dec930c50f52.png)|
|Site Kit Activated (input is disabled)|![image](https://user-images.githubusercontent.com/22185279/104629659-00c53600-5657-11eb-8042-3df0800becd3.png)|


---



Fixes #5877
